src/test/java/core/basesyntax/service/impl/ReportMakerServiceImplTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportMakerService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class ReportMakerServiceImplTest {
+    private final ReportMakerService reportMakerService = new ReportMakerServiceImpl();
+
+    @AfterEach
```

Comment:
better to use beforeEach because there could be data it storage before you start test